site/app/views/email/EmailStatusView.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace app\views\email;
+
+use app\views\AbstractView;
+use app\models\User;
+use app\libraries\FileUtils;
+use app\models\EmailStatusModel;
+
+class EmailStatusView extends AbstractView {
+    public function showEmailStatus($emailStatus) {
+        $this->core->getOutput()->addBreadcrumb("Email", $this->core->buildCourseUrl(["email"]));
+        $this->core->getOutput()->addVendorJs('bootstrap/js/bootstrap.bundle.min.js');
+        $this->core->getOutput()->addInternalCss('bootstrap.css');
+        $this->core->getOutput()->addInternalCss('email-status.css');
+        $this->core->getOutput()->addInternalJs('email-status.js');
```

Comment:
I don't see any file that has this name?

Reply:
I moved the script to the twig file and deleted the file, but it seems I forgot to remove the addInternalJs line, sorry! However, the email-status.css file should be in public/css, was this file in there for you?